hack/kind-cluster.sh

```diff
@@ -32,5 +34,28 @@ nodes:
         extraArgs:
             v: "5"
 EOF
+}
+
+install_operator() {
+  pushd ${DIR}/../
+  make install
+  make docker-build IMG=localhost/ingress-node-firewall-controller:local
+  kind load docker-image localhost/ingress-node-firewall-controller:local
+  make docker-build-daemon DAEMON_IMG=localhost/ingress-node-firewall-daemon:local
+  kind load docker-image localhost/ingress-node-firewall-daemon:local
+  if ! command -v yq ; then 
+      echo "Cannot find command yq."
+      echo "Please modify file config/manager/env.yaml manually and point DAEMONSET_IMAGE to:"
+      echo "    localhost/ingress-node-firewall-daemon:local"
+      echo "Then, hit any key"
+      read
+  fi
+  yq e '.spec.template.spec.containers[0].env[0].value = "localhost/ingress-node-firewall-daemon:local"'  -i config/manager/env.yaml
```

Comment:
hard coding container idx and env idx to match env.yaml will cause trouble if we reorder anything in env.yaml by accident can we find the index by lookup the env name ?